Sming/Wiring/HexString.cpp

```diff
@@ -0,0 +1,29 @@
+/****
```

Comment:
Can you move the HexString.* files to SmingCore/Data/ ? I would prefer to keep in the Wiring directory mostly classes coming from and/or improving [Wiring](http://wiring.org.co/).